app/facades/quote_facade.rb

```diff
@@ -0,0 +1,9 @@
+class QuoteFacade
+
+  def self.quote_finder(keyword)
+    quotes_data = QuoteService.quotes_search(keyword)
+    quotes_data[:results].map do |quote|
+      Quote.new(quote)
+    end
+  end
```

Comment:
👍 I mentioned earlier that you missed getting the totalCount from the response. Here, you could get it from the quotes_data. `quotes_data[:totalCount]`

Reply:
I'd probably limit the quotes that get returned here rather than doing it in your view. That way, you're not sending more to the view than you need.